app/filter-helpers.js

```diff
@@ -10,19 +10,60 @@ for (var group in formatMappingData) {
 
 var sectionFilter = function (allContent, pageSection) {
   console.log("Filtering " + allContent.length + " items by " + pageSection);
+  var filteredList = _filteredList(allContent, pageSection);
+
+  console.log("Filtered out " + filteredList.length);
+  console.log(filteredList);
+
+  return filteredList;
+}
+
+var rowsWithThreeColumns = function(allContent, pageSection) {
+  var filteredList = _filteredList(allContent, pageSection);
+
+  var rows    = { rows: [] };
```

Comment:
I can see the naming difficulty here, but this would be better off called something like `rowWrapper`. Saves you having to write `rows.rows` below.